src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -663,6 +663,11 @@ public string PythonTemplateFilePath
         /// </summary>
         public bool EnableNodeAutoComplete { get; set; }
 
+        /// <summary>
+        /// Enable New UI for cluster node autocomplete
+        /// </summary>
+        public bool MLEnableNewUI { get; set; }
```

Comment:
Ideally we should not add anything to the api related to this. This new button will be obsolete once the new experience becomes the default one and we retire the other one.